app/workflows/lists/conversion/voluntary/sections/ready-for-opening.yml

```diff
@@ -7,7 +7,7 @@ tasks:
       the Baseline email. Indicate on the Baseline that the school is on track
       to open.
     actions:
-      - title: Confirm the school is on track to open on the target date
+      - title: Confirm the school is on track to open on the provisional date
```

Comment:
The previous commit swaps these over. I'd drop the previous commit as I assume it's not needed.